### ISSUE TITLE
Accessibility: Timeline view events are not focusable 

### ISSUE THREAD

**Issue by zac1st1k** [fullcalendar/fullcalendar#7571]
### Reduced Test Case

https://fullcalendar.io/demos

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

No connector (vanilla JS)

### Bug Description

The events in timeline view cannot gain focus.

<img width="1079" alt="image" src="https://github.com/fullcalendar/fullcalendar/assets/1715609/667a3f50-02de-468f-8876-19cf0de1c1a3">

**Comment by acerix** [comment]
I believe this existing ticket captures this need:
https://github.com/fullcalendar/fullcalendar/issues/4513
↑
Please visit the above issue, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates.

**Comment by zac1st1k** [comment]
Hi @acerix ,

https://github.com/fullcalendar/fullcalendar/issues/4513 is to add touch screen support?

This ticket is to add keyboard navigation support. They are not the same?

Thanks for reviewing the ticket.

**Comment by acerix** [comment]
If you mean selecting event links by pressing "tab" you can do that if you set a URL to make the events a link.  There is also this open issue for interacting with events by keyboard:

https://github.com/fullcalendar/fullcalendar/issues/2535

**Comment by Malinovavi** [comment]
I still don't think this is the same issue as 2535. With a screen reader such as NVDA, I should be able to navigate using the arrows on my keyboard and have the SR announce the events in the table even if the events are not interactive. There is a loss of information making the timeline plugin not accessible

![Image](https://github.com/user-attachments/assets/95f04a61-67c9-4206-8eec-b5cb094cfdae)
